### ISSUE TITLE
Aliases VS Code Extension Wishlist

### ISSUE THREAD

**Issue by EddyLuten** [EddyLuten/mkdocs-alias-plugin#2]
It would be nice to have a tie-in VS Code extension that added first-class support for aliases in Markdown files. I'm capturing some of the requirements here before starting development on it.

## Proper Rendering

Aliases are rendered strangely within VS Code:

<img width="260" alt="Screen Shot 2022-07-07 at 11 11 07 AM" src="https://user-images.githubusercontent.com/1082806/177820904-03fae1a2-b99e-4a6f-98f9-f578076c3fdc.png">

Rendering these aliases closer to links would be preferable:

<img width="399" alt="Screen Shot 2022-07-07 at 11 13 52 AM" src="https://user-images.githubusercontent.com/1082806/177821404-c588d9e9-54f4-4d26-9de7-2a199dc7989e.png">

Also perhaps displaying red squigglies if the alias wasn't found within the scope of the project.

## Auto-Completion

While typing the `[[` alias syntax, it would be nice to have an auto-complete dialog show with known aliases and their associated page titles.

<img width="880" alt="Screen Shot 2022-07-07 at 11 16 43 AM" src="https://user-images.githubusercontent.com/1082806/177821899-d4154b97-6805-4839-9c46-1f13ff72b00f.png">

## Defining Aliases

It would be nice to have a way to easily define an alias for any active Markdown file, including inserting a meta-section into the file if none exists.
